lib/audit_help.c

```diff
@@ -44,6 +45,23 @@ void audit_help_open (void)
 	}
 }
 
+/*
+ * This takes a string and replaces the old character with the new.
+ */
+static inline char *strtr (char *str, char old, char new)
+{
+	if (str == NULL) {
+		return NULL;
+	}
+
+	for (char *p = str; *p; p++) {
```

Comment:
Please use `streq(p, "")` instead of `*p` for the condition.

Using a string API means that it would be possible to add _FORTIFY_SOURCE fortification to the pointer access.